templates/web/peterborough/waste/landlord_info.html

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-warning-text due">
+    <div class="govuk-warning-text__img">
+        <span class="govuk-warning-text__icon" aria-hidden="true">i</span>
+    </div>
+    <div class="govuk-warning-text__content">
+        <span class="govuk-warning-text__assistive">Important information</span>
+        <p class="govuk-!-margin-bottom-0">If you are a Landlord or property manager please contact 01733 747474 to request a quote.</p>
+        <!--
```

Comment:
Does this commented section need to be left in?

Reply:
I left it in there just in case we want to re-use it with a button, instead of just text. But we can get rid of it we won't use it.